bin/generate_label.py

```diff
@@ -17,14 +14,22 @@
 import tifffile as tf
 import pandas as pd
 from pathlib import Path
-from process_spaceranger import spaceranger_to_h5ad
+from process_spaceranger import spaceranger_to_anndata
+
 
+def main(stem: str, ome_md: dict[str, str], h5ad: str) -> None:
+    """This function writes a label image of spots
```

Comment:
Should we specify this is visium spot exclusively?

Reply:
@dannda has already updated these doc strings in the newer pull request (#56), so let's defer to those updates for this discussion. It could be worth clarifying the function is currently used exclusively for Visium, although the function would work for any anndata that contained the correct data, so it could also be worth linking to the expected spatial schema in the docs too. Let's pick up the disucssion in #56.